tools/graphmap/macros.xml

```diff
@@ -0,0 +1,23 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+        <requirement type="package" version="0.5.2">graphmap</requirement>
```

Comment:
You could make the version number a macro and use it as well in the tool-version.

Reply:
:+1: